source/_docs/domains.md

```diff
@@ -139,16 +139,36 @@ The same technique works for single subdomain redirects. Just specify the path i
 
 #### Redirect From One Path to Another
 
+For a single path:
```

Comment:
@alexfornuto can you add stronger intros to help contextualize the usage of the snippet?